experiments/AMIP/moist_mpi_earth/coupler_driver.jl

```diff
@@ -114,37 +150,37 @@ if mode_name == "amip"
             SST_info = SST_info,
         )
     )
-    # 3. sea ice
+    ## sea ice
     SIC_info = bcfile_info_init(
         FT,
         comms_ctx,
         sic_data,
         "SEAICE",
         boundary_space,
         interpolate_daily = true,
-        scaling_function = clean_sic, # convert to fractions
+        scaling_function = clean_sic, ## convert to fractions
         land_mask = land_mask,
         date0 = date0,
         mono = mono_surface,
     )
     update_midmonth_data!(date0, SIC_info)
     SIC_init = interpolate_midmonth_to_daily(date0, SIC_info)
-    ice_mask = get_ice_mask.(SIC_init, mono_surface)
+    ice_mask = get_ice_mask.(SIC_init, mono_surface) .* FT(0)
```

Comment:
Thank you for the review! Yeah, this was done to let the system spin up ice-free for the first 15 days. The mask then gets updated with the prescribed SIC thereafter. That said, after the latest changes in surface fluxes, this may not make that much of a difference. I can take a look. :)